test/src/core/SceneRunnerCSharpSceneTest.cs

```diff
@@ -71,6 +71,16 @@ public void LoadSceneBinaryFormat()
             .IsNotNull();
     }
 
+
+    [TestCase]
+    public void LoadSceneNode()
+    {
+        using var runner = ISceneRunner.Load(new Node(), true);
+        AssertThat(runner.Scene())
+            .IsInstanceOf<Node>()
+            .IsNotNull();
```

Comment:
Instead of checking is not null, should be checking the scene has the same instance as the Load input argument.